docs/source/examples/convert_particle_stacks.rst

```diff
@@ -1,48 +1,37 @@
 Convert Particles
 =================
 
-.. code-block:: python
+This script demonstrates the usage of the
```

Comment:
I simplified this (and the other examples) so that: it has only one copy of the text, the text is not treated as part of a code block, and the actual python code is imported rather than being copy-pasted.